ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -690,10 +682,12 @@ keys:
           type: str
           convert_types:
           - bool
+          valid_values: ["enabled", "disabled", "true"]
         spanning_tree_bpduguard:
           type: str
           convert_types:
           - bool
+          valid_values: ["enabled", "disabled", "true"]
```

Comment:
I think you might need to also have `"True"` in here, since the type conversion `str(True)` -> `"True"`

Reply:
same above